pkg/database/dataproducts.go

```diff
@@ -108,6 +107,13 @@ func (r *Repo) CreateDataproduct(ctx context.Context, dp models.NewDataproduct,
 		return nil, err
 	}
 
+	if err := r.CreateTeamProductAreaMappingIfNotExists(ctx, tx, dp.TeamID, dp.ProductAreaID); err != nil {
```

Comment:
Is it really necessary to update the mapping case by case, when we already have the update routine? I know you only update the teams that are already in our database for now, but what if we fetch all teams from teamkatalogen, and "upsert" into our mapping? I feel the code could be simpler, and faster (less rest calls), as long as we do not have too many teams in teamkatalogen, which is the case for foreseeable future

Reply:
I agree, will change!